src/ast/prototype_ast.hpp

```diff
@@ -15,6 +15,7 @@
 class PrototypeAST
 {
     std::string Name;
+    //add return type in future
```

Comment:
```suggestion
```

I don't think this comment provides enough context to be actionable in the future. It should be removed before merging.

Reply:
Would a more descriptive comment make sense here, or should I just refrain from adding any comments like this when I merge branches into main?